data.sql

```diff
@@ -1,5 +1,19 @@
-/* Populate database with sample data. */
-
-INSERT INTO animals (name) VALUES ('Luna');
-INSERT INTO animals (name) VALUES ('Daisy');
-INSERT INTO animals (name) VALUES ('Charlie');
+/* Day 1 use of queries */
+INSERT INTO animals (name, date_of_birth, escape_attempt, neutered, weight_kg)
```

Comment:
- [ ] [OPTIONAL] I kindly suggest that you change the column `escape_attempt` to become plural because the attempt might be more than one